msgid ""
msgstr ""
"Language: it-IT\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Phrase (phrase.com)\n"

#: pydantic_translations/_messages.py:58
#, python-brace-format
msgid "{value} is not a valid Enum instance"
msgstr "{value} non è un'istanza valida di Enum"

#: pydantic_translations/_messages.py:63
#, python-brace-format
msgid "{value} is not a valid IntEnum instance"
msgstr "{value} non è un'istanza valida di IntEnum"

#: pydantic_translations/_messages.py:52
#, python-brace-format
msgid "{value} is not callable"
msgstr "{value} non è di tipo callable"

#: pydantic_translations/_messages.py:53
msgid "a class is expected"
msgstr "è prevista una classe"

#: pydantic_translations/_messages.py:18
msgid "byte type expected"
msgstr "tipo di byte previsto"

#: pydantic_translations/_messages.py:107
msgid "card number is not all digits"
msgstr "il numero della carta non è composto solo da cifre"

#: pydantic_translations/_messages.py:109
msgid "card number is not luhn valid"
msgstr "il numero della carta non è valido in Luhn"

#: pydantic_translations/_messages.py:93
#, python-brace-format
msgid "could not interpret byte unit: {unit}"
msgstr "impossibile interpretare l'unità di byte: {unit}"

#: pydantic_translations/_messages.py:23
msgid "could not parse value and unit from byte string"
msgstr "impossibile analizzare il valore e l'unità dalla stringa di byte"

#: pydantic_translations/_messages.py:82
msgid "date is not in the future"
msgstr "la data non è nel futuro"

#: pydantic_translations/_messages.py:83
msgid "date is not in the past"
msgstr "la data non è nel passato"

#: pydantic_translations/_messages.py:89
#, python-brace-format
msgid "Discriminator {discriminator_key!r} is missing in value"
msgstr "Il discriminatore {discriminator_key!r} è privo di valore"

#: pydantic_translations/_messages.py:85
#, python-brace-format
msgid "ensure that there are no more than {decimal_places} decimal places"
msgstr "verificare che non ci siano più di {decimal_places} cifre decimali"

#: pydantic_translations/_messages.py:84
#, python-brace-format
msgid "ensure that there are no more than {max_digits} digits in total"
msgstr "verificare che non ci siano più di {max_digits} cifre in totale"

#: pydantic_translations/_messages.py:87
#, python-brace-format
msgid "ensure that there are no more than {whole_digits} digits before the decimal point"
msgstr "verificare che non ci siano più di {whole_digits} cifre prima della virgola decimale"

#: pydantic_translations/_messages.py:72
#, python-brace-format
msgid "ensure this value contains valid import path or valid callable: {error_message}"
msgstr "verificare che questo valore contenga un percorso di importazione valido o un callable valido: {error_message}"

#: pydantic_translations/_messages.py:79
#, python-brace-format
msgid "ensure this value has at least {limit_value} characters"
msgstr "verificare che questo valore contenga almeno {limit_value} caratteri"

#: pydantic_translations/_messages.py:112
#, python-brace-format
msgid "ensure this value has at least {limit_value} items"
msgstr "verificare che questo valore contenga almeno {limit_value} elementi"

#: pydantic_translations/_messages.py:78
#, python-brace-format
msgid "ensure this value has at most {limit_value} characters"
msgstr "verificare che questo valore contenga al massimo {limit_value} caratteri"

#: pydantic_translations/_messages.py:111
#, python-brace-format
msgid "ensure this value has at most {limit_value} items"
msgstr "verificare che questo valore contenga al massimo {limit_value} elementi"

#: pydantic_translations/_messages.py:98
msgid "ensure this value is a finite number"
msgstr "verificare che questo valore sia un numero finito"

#: pydantic_translations/_messages.py:103
#, python-brace-format
msgid "ensure this value is a multiple of {multiple_of}"
msgstr "verificare che questo valore sia un multiplo di {multiple_of}"

#: pydantic_translations/_messages.py:100
#, python-brace-format
msgid "ensure this value is greater than {limit_value}"
msgstr "verificare che questo valore sia maggiore di {limit_value}"

#: pydantic_translations/_messages.py:99
#, python-brace-format
msgid "ensure this value is greater than or equal to {limit_value}"
msgstr "verificare che questo valore sia maggiore o uguale a {limit_value}"

#: pydantic_translations/_messages.py:102
#, python-brace-format
msgid "ensure this value is less than {limit_value}"
msgstr "verificare che questo valore sia inferiore a {limit_value}"

#: pydantic_translations/_messages.py:101
#, python-brace-format
msgid "ensure this value is less than or equal to {limit_value}"
msgstr "verificare che questo valore sia inferiore o uguale a {limit_value}"

#: pydantic_translations/_messages.py:90
msgid "extra fields not permitted"
msgstr "campi aggiuntivi non consentiti"

#: pydantic_translations/_messages.py:97
msgid "field required"
msgstr "campo obbligatorio"

#: pydantic_translations/_messages.py:106
#, python-brace-format
msgid "file or directory at path \"{path}\" does not exist"
msgstr "Il file o la directory nel percorso \"{path}\" non esiste."

#: pydantic_translations/_messages.py:54
#, python-brace-format
msgid "instance of {class_name}, tuple or dict expected"
msgstr "istanza di {class_name}, tupla o dict prevista"

#: pydantic_translations/_messages.py:50
#, python-brace-format
msgid "instance of {expected_arbitrary_type} expected"
msgstr "istanza di {expected_arbitrary_type} prevista"

#: pydantic_translations/_messages.py:24
msgid "invalid date format"
msgstr "formato data non valido"

#: pydantic_translations/_messages.py:25
msgid "invalid datetime format"
msgstr "formato data-ora non valido"

#: pydantic_translations/_messages.py:26
msgid "invalid duration format"
msgstr "formato di durata non valido"

#: pydantic_translations/_messages.py:27
msgid "Invalid JSON"
msgstr "JSON non valido"

#: pydantic_translations/_messages.py:118
msgid "invalid or missing URL scheme"
msgstr "schema URL non valido o mancante"

#: pydantic_translations/_messages.py:110
msgid "Invalid regular expression"
msgstr "Espressione regolare non valida"

#: pydantic_translations/_messages.py:28
msgid "invalid time format"
msgstr "formato orario non valido"

#: pydantic_translations/_messages.py:66
msgid "JSON object must be str, bytes or bytearray"
msgstr "L'oggetto JSON deve essere str, byte o bytearray"

#: pydantic_translations/_messages.py:108
#, python-brace-format
msgid "Length for a {brand} card must be {required_length}"
msgstr "La lunghezza di una carta {brand} deve essere {required_length}"

#: pydantic_translations/_messages.py:88
#, python-brace-format
msgid "No match for discriminator {discriminator_key!r} and value {discriminator_value!r} (allowed values: {allowed_values})"
msgstr "Nessuna corrispondenza per il discriminatore {discriminator_key!r} e il valore {discriminator_value!r} (valori ammessi: {allowed_values})"

#: pydantic_translations/_messages.py:69
msgid "none is not an allowed value"
msgstr "\"none\" non è un valore consentito"

#: pydantic_translations/_messages.py:104
#, python-brace-format
msgid "path \"{path}\" does not point to a directory"
msgstr "il percorso \"{path}\" non punta a una directory"

#: pydantic_translations/_messages.py:105
#, python-brace-format
msgid "path \"{path}\" does not point to a file"
msgstr "il percorso \"{path}\" non punta a un file"

#: pydantic_translations/_messages.py:19
msgid "str type expected"
msgstr "tipo str previsto"

#: pydantic_translations/_messages.py:113
#, python-brace-format
msgid "string does not match regex \"{pattern}\""
msgstr "la stringa non corrisponde alla regex \"{pattern}\"."

#: pydantic_translations/_messages.py:75
#, python-brace-format
msgid "subclass of {expected_class} expected"
msgstr "sottoclasse di {expected_class} prevista"

#: pydantic_translations/_messages.py:96
msgid "the list has duplicated items"
msgstr "l'elenco ha voci duplicate"

#: pydantic_translations/_messages.py:81
#, python-brace-format
msgid "unexpected value; permitted: {permitted}"
msgstr "valore inatteso; consentito: {permitted}"

#: pydantic_translations/_messages.py:116
msgid "URL host invalid"
msgstr "URL host non valido"

#: pydantic_translations/_messages.py:116
msgid "URL host invalid, top level domain required"
msgstr "URL host non valido, è necessario un dominio di primo livello"

#: pydantic_translations/_messages.py:115
#, python-brace-format
msgid "URL invalid, extra characters found after valid URL: {extra!r}"
msgstr "URL non valido, caratteri in eccesso trovati dopo l'URL valido: {extra!r}"

#: pydantic_translations/_messages.py:117
msgid "URL port invalid, port cannot exceed 65535"
msgstr "Porta URL non valida, la porta non può essere superiore a 65535"

#: pydantic_translations/_messages.py:118
msgid "URL scheme not permitted"
msgstr "Schema URL non consentito"

#: pydantic_translations/_messages.py:119
msgid "userinfo required in URL but missing"
msgstr "l’URL richiede le info utente mancanti"

#: pydantic_translations/_messages.py:120
#, python-brace-format
msgid "uuid version {required_version} expected"
msgstr "prevista versione uuid {required_version}"

#: pydantic_translations/_messages.py:20
msgid "value could not be parsed to a boolean"
msgstr "non può essere analizzato in un valore booleano"

#: pydantic_translations/_messages.py:51
msgid "value is not a valid boolean"
msgstr "il valore non è un booleano valido"

#: pydantic_translations/_messages.py:80
#, python-brace-format
msgid "value is not a valid color: {reason}"
msgstr "il valore non è un colore valido: {reason}"

#: pydantic_translations/_messages.py:86
msgid "value is not a valid decimal"
msgstr "il valore non è un decimale valido"

#: pydantic_translations/_messages.py:56
msgid "value is not a valid deque"
msgstr "il valore non è un deque valido"

#: pydantic_translations/_messages.py:57
msgid "value is not a valid dict"
msgstr "il valore non è un dict valido"

#: pydantic_translations/_messages.py:29
msgid "value is not a valid email address"
msgstr "il valore non è un indirizzo email valido"

#: pydantic_translations/_messages.py:59
#, python-brace-format
msgid "value is not a valid enumeration member; permitted: {permitted}"
msgstr "il valore non è un membro valido dell'enumerazione; consentito: {permitted}"

#: pydantic_translations/_messages.py:60
msgid "value is not a valid float"
msgstr "il valore non è un fluttuante valido"

#: pydantic_translations/_messages.py:61
msgid "value is not a valid frozenset"
msgstr "il valore non è un frozenset valido"

#: pydantic_translations/_messages.py:62
msgid "value is not a valid hashable"
msgstr "il valore non è un hashable valido"

#: pydantic_translations/_messages.py:64
msgid "value is not a valid integer"
msgstr "il valore non è un numero intero valido"

#: pydantic_translations/_messages.py:30
msgid "value is not a valid IPv4 address"
msgstr "il valore non è un indirizzo IPv4 valido"

#: pydantic_translations/_messages.py:31
msgid "value is not a valid IPv4 interface"
msgstr "il valore non è un'interfaccia IPv4 valida"

#: pydantic_translations/_messages.py:32
msgid "value is not a valid IPv4 network"
msgstr "il valore non è una rete IPv4 valida"

#: pydantic_translations/_messages.py:33
msgid "value is not a valid IPv4 or IPv6 address"
msgstr "il valore non è un indirizzo IPv4 o IPv6 valido"

#: pydantic_translations/_messages.py:34
msgid "value is not a valid IPv4 or IPv6 interface"
msgstr "il valore non è un'interfaccia IPv4 o IPv6 valida"

#: pydantic_translations/_messages.py:35
msgid "value is not a valid IPv4 or IPv6 network"
msgstr "il valore non è una rete IPv4 o IPv6 valida"

#: pydantic_translations/_messages.py:36
msgid "value is not a valid IPv6 address"
msgstr "il valore non è un indirizzo IPv6 valido"

#: pydantic_translations/_messages.py:37
msgid "value is not a valid IPv6 interface"
msgstr "il valore non è un'interfaccia IPv6 valida"

#: pydantic_translations/_messages.py:38
msgid "value is not a valid IPv6 network"
msgstr "il valore non è una rete IPv6 valida"

#: pydantic_translations/_messages.py:65
msgid "value is not a valid iterable"
msgstr "il valore non è un iterabile valido"

#: pydantic_translations/_messages.py:67
msgid "value is not a valid list"
msgstr "il valore non è un elenco valido"

#: pydantic_translations/_messages.py:71
msgid "value is not a valid path"
msgstr "il valore non è un percorso valido"

#: pydantic_translations/_messages.py:73
msgid "value is not a valid sequence"
msgstr "il valore non è una sequenza valida"

#: pydantic_translations/_messages.py:74
msgid "value is not a valid set"
msgstr "il valore non è un set valido"

#: pydantic_translations/_messages.py:76
msgid "value is not a valid tuple"
msgstr "il valore non è una tupla valida"

#: pydantic_translations/_messages.py:77
msgid "value is not a valid uuid"
msgstr "il valore non è un uuid valido"

#: pydantic_translations/_messages.py:68
msgid "value is not none"
msgstr "il valore non è none"

#: pydantic_translations/_messages.py:70
msgid "value is not None"
msgstr "il valore non è None"

#: pydantic_translations/_messages.py:114
#, python-brace-format
msgid "wrong tuple length {actual_length}, expected {expected_length}"
msgstr "lunghezza tupla errata {actual_length}, prevista {expected_length}"
